From: Henry Su <henry.su@amd.com>
Date: Thu, 24 May 2007 00:42:37 +0000 (+0200)
Subject: add the IDE device ID for ATI SB700
X-Git-Tag: v2.6.22-rc3~89
X-Git-Url: http://git.kernel.org/?p=linux%2Fkernel%2Fgit%2Ftorvalds%2Flinux-2.6.git;a=commitdiff_plain;h=6c6a2a8d201b4f8fd54167802da5ddbe08abd744

add the IDE device ID for ATI SB700

Add the IDE device ID to atiixp_pci_tbl struct in atiixp.c for ATI SB700.

From: Henry Su <henry.su@amd.com>
Signed-off-by: Luugi Marsan <luugi.marsan@amd.com>
Signed-off-by: Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
---

Backported to Debian's 2.6.18 by dann frazier <dannf@debian.org>

diff -urpN linux-source-2.6.18.orig/drivers/ide/pci/atiixp.c linux-source-2.6.18/drivers/ide/pci/atiixp.c
--- linux-source-2.6.18.orig/drivers/ide/pci/atiixp.c	2007-08-03 01:28:24.135117000 -0600
+++ linux-source-2.6.18/drivers/ide/pci/atiixp.c	2007-08-03 01:48:49.152931194 -0600
@@ -365,6 +365,7 @@ static struct pci_device_id atiixp_pci_t
 	{ PCI_VENDOR_ID_ATI, PCI_DEVICE_ID_ATI_IXP300_IDE, PCI_ANY_ID, PCI_ANY_ID, 0, 0, 0},
 	{ PCI_VENDOR_ID_ATI, PCI_DEVICE_ID_ATI_IXP400_IDE, PCI_ANY_ID, PCI_ANY_ID, 0, 0, 0},
 	{ PCI_VENDOR_ID_ATI, PCI_DEVICE_ID_ATI_IXP600_IDE, PCI_ANY_ID, PCI_ANY_ID, 0, 0, 1},
+	{ PCI_VENDOR_ID_ATI, PCI_DEVICE_ID_ATI_IXP700_IDE, PCI_ANY_ID, PCI_ANY_ID, 0, 0, 1},
 	{ 0, },
 };
 MODULE_DEVICE_TABLE(pci, atiixp_pci_tbl);
diff -urpN linux-source-2.6.18.orig/include/linux/pci_ids.h linux-source-2.6.18/include/linux/pci_ids.h
--- linux-source-2.6.18.orig/include/linux/pci_ids.h	2007-08-03 01:44:58.149571000 -0600
+++ linux-source-2.6.18/include/linux/pci_ids.h	2007-08-03 01:48:49.152931194 -0600
@@ -364,6 +364,7 @@
 #define PCI_DEVICE_ID_ATI_IXP600_SRAID	0x4381
 #define PCI_DEVICE_ID_ATI_IXP600_IDE	0x438c
 #define PCI_DEVICE_ID_ATI_IXP700_SATA	0x4390
+#define PCI_DEVICE_ID_ATI_IXP700_IDE	0x439c
 
 #define PCI_VENDOR_ID_VLSI		0x1004
 #define PCI_DEVICE_ID_VLSI_82C592	0x0005
